data/association-flag-map.yaml

```diff
@@ -79,18 +79,18 @@ columns:
     - name: slot_Shape_flag
       bit: 23
       doc: General Failure Flag
-    - name: slot_Shape_flag_unweightedBad
-      bit: 24
-      doc: Both weighted and unweighted moments were invalid
-    - name: slot_Shape_flag_unweighted
-      bit: 25
-      doc: Weighted moments converged to an invalid value; using unweighted moments
-    - name: slot_Shape_flag_shift
-      bit: 26
-      doc: centroid shifted by more than the maximum allowed amount
-    - name: slot_Shape_flag_maxIter
-      bit: 27
-      doc: Too many iterations in adaptive moments
-    - name: slot_Shape_flag_psf
-      bit: 28
-      doc: Failure in measuring PSF model shape
+#    - name: slot_Shape_flag_unweightedBad
```

Comment:
Please add a comment above this block about why all of these were commented out.

Reply:
I've removed all of the SdssShape values instead of commenting them out, and replaced them with the flags from HsmShape.cc.